src/schema/schema_create.c

```diff
@@ -433,6 +433,57 @@ __wt_find_import_metadata(WT_SESSION_IMPL *session, const char *uri, const char
     return (0);
 }
 
+/*
+ * __schema_is_tiered_storage_shared --
+ *     Check whether the table is configured for a tiered storage shared.
+ */
+static void
+__schema_is_tiered_storage_shared(
+  WT_SESSION_IMPL *session, const char *config, bool *is_tiered_storage_shared)
+{
+    WT_CONFIG_ITEM cval;
+
+    *is_tiered_storage_shared = true;
+
```

Comment:
There are a lot of complex conditionals here. I think this needs a comment explaining what you're checking for and why.

Reply:
Added a comment to explain why we cannot create tiered storage shared in those create configurations because of the control of the underlying source.